### DISCUSSION TITLE
Change the lock screen wallpaper when switching from one mode to another

### DISCUSSION THREAD

**Discussion by juancholopes** [AutoDarkMode/Windows-Auto-Night-Mode#885] [Custom scripts]
I realized that it would be possible to create a script that not only changes the desktop wallpaper when switching between light mode and dark mode, but also allows us to customize the lock screen background to whatever we desire. Has anyone already done this?

**Comment by YounessX3** [comment]
Any news?

**Reply by Armin2208** [reply]
No, nothing

**Comment by PastLives-GIT** [comment]
Try my script: [https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/discussions/1254#discussion-10645238](url) (In discussion: Set lock screen wallpaper)

**Reply by Armin2208** [reply]
Link is broken, so here the fixed: [https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/discussions/1254](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/discussions/1254 )
